cocos/ui/UIEditBox/UIEditBoxImpl-common.h

```diff
@@ -133,6 +136,14 @@ class EditBoxImplCommon : public EditBoxImpl
     
     int   _maxLength;
     Size _contentSize;
+    
```

Comment:
@zilongshanren  If I remove data from EditBox.h then How to access the private data of EditBox-common.h in EditBox.cpp.

For Ex I see
InputMode    _editBoxInputMode;
InputFlag    _editBoxInputFlag;
EditBox::KeyboardReturnType  _keyboardReturnType;
Color4B _colText;
Color4B _colPlaceHolder;

These value are also duplicate in EditBox.h and EditBox-common.h